### ISSUE TITLE
Decide on selection colors wrt web accessibility

### ISSUE THREAD

**Issue by eileen-kuehn** [SORSE/sorse.github.io#47]
We tried a selection background color of ``#CCC`` with font color ``#FFF`` but if [fails with respect to contrast for web accessibility with a ratio of 1.6:1](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=CCCCCC). We currently use background ``#000`` and foreground. ``#FFF`` (ratio: 21:1). This works for now but could be improved to better match the template.

However, I think this is on low priority :)

_Originally posted by @eileen-kuehn in https://github.com/DE-RSE/SORSE20/pull/43#issuecomment-641173295_

**Comment by Chilipp** [comment]
@eileen-kuehn can this actually be closed? (see #167)

**Comment by Chilipp** [comment]
thanks @eileen-kuehn! Actually I realized that this issue was about the color of the text selection, and this has not been addressed by the layout changes :sweat_smile: That's why I deleted my previous comment directly after I did post it here

**Comment by eileen-kuehn** [comment]
@Chilipp, yes, the title of the ticket was very specific. So it should better have been something as picking template... ;). In any case, thanks for taking care about the old tickets!
